<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>707</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Camera Calibration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="main_layout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="playback_area">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="playback_widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="playback_display_widget_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" name="playback_display_layout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="playback_display">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Drag and drop into this window
Or go to File &gt; Open</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="playback_control_layout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QPushButton" name="to_beg_button">
             <property name="text">
              <string>To Beginning</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="prev_bd_button">
             <property name="text">
              <string>Prev Board</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="prev_frame_button">
             <property name="text">
              <string>Prev Frame</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="play_button">
             <property name="text">
              <string>Play</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="next_frame_button">
             <property name="text">
              <string>Next Frame</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="next_bd_button">
             <property name="text">
              <string>Next Board</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="to_end_button">
             <property name="text">
              <string>To End</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="sidebar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="cb_settings_layout">
           <property name="title">
            <string>Chessboard settings</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="board_width_layout">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="board_width_lbl">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Width (#corners):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="board_width_edit">
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="board_height_layout">
              <item>
               <widget class="QLabel" name="board_height_lbl">
                <property name="text">
                 <string>Height (#corners):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="board_height_edit">
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="display_board_button">
              <property name="text">
               <string>Display board</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="cam_settings_layout">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Camera settings</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="cam_name_layout">
              <item>
               <widget class="QLabel" name="cam_name_lbl">
                <property name="text">
                 <string>Camera name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cam_name_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>107</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Camera</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="sensor_width_layout">
              <item>
               <widget class="QLabel" name="sensor_width_lbl">
                <property name="text">
                 <string>Sensor width (mm):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="sensor_width_edit">
                <property name="minimum">
                 <double>0.100000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>36.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="calibration_layout">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Calibration</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="cov_disp_layout">
              <item>
               <widget class="QLabel" name="tot_cov_lbl">
                <property name="minimumSize">
                 <size>
                  <width>122</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Total coverage (%):</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tot_cov_num">
                <property name="text">
                 <string>0.0000</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="update_solution_button">
              <property name="text">
               <string>Update solution</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="res_layout">
              <item>
               <layout class="QVBoxLayout" name="res_var_lbls">
                <item>
                 <widget class="QLabel" name="avg_err_lbl">
                  <property name="text">
                   <string>Avg. error (px):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="sol_cov_lbl">
                  <property name="text">
                   <string>Solution coverage (%):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="focal_length_lbl">
                  <property name="text">
                   <string>Focal length (mm):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="hfov_lbl">
                  <property name="text">
                   <string>hFOV (deg):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="norm_fx_lbl">
                  <property name="text">
                   <string>Norm fx:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dist_k1_lbl">
                  <property name="text">
                   <string>K1:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dist_k2_lbl">
                  <property name="text">
                   <string>K2:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dist_k3_lbl">
                  <property name="text">
                   <string>K3:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="res_var_nums">
                <item>
                 <widget class="QLabel" name="avg_err_num">
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::AutoText</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="sol_cov_num">
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="focal_length_num">
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="hfov_num">
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="norm_fx_num">
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dist_k1_num">
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dist_k2_num">
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dist_k3_num">
                  <property name="text">
                   <string>0.0000</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="export_prof_button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Export Camera Profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPrevious_frame"/>
    <addaction name="actionNext_frame"/>
    <addaction name="actionPrevious_board"/>
    <addaction name="actionNext_board"/>
    <addaction name="actionJump_to_beginning"/>
    <addaction name="actionJump_to_end"/>
    <addaction name="separator"/>
    <addaction name="actionDetect_board_on_current_frame"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNext_frame">
   <property name="text">
    <string>Next frame</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious_frame">
   <property name="text">
    <string>Previous frame</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNext_board">
   <property name="text">
    <string>Next board</string>
   </property>
   <property name="shortcut">
    <string>Shift+Right</string>
   </property>
  </action>
  <action name="actionPrevious_board">
   <property name="text">
    <string>Previous board</string>
   </property>
   <property name="shortcut">
    <string>Shift+Left</string>
   </property>
  </action>
  <action name="actionJump_to_end">
   <property name="text">
    <string>Jump to end</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionJump_to_beginning">
   <property name="text">
    <string>Jump to beginning</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionDetect_board_on_current_frame">
   <property name="text">
    <string>Detect board on current frame</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
